codebuild_specs/e2e_workflow.yml

```diff
@@ -521,10 +511,20 @@ batch:
         compute-type: BUILD_GENERAL1_SMALL
         variables:
           TEST_SUITE: src/__tests__/transformer-migrations/searchable-migration.test.ts
-          CLI_REGION: us-east-2
+          CLI_REGION: us-east-1
           USE_PARENT_ACCOUNT: 1
       depend-on:
         - publish_to_local_registry
+    - identifier: add_resources_base_cdk_custom_logic_relationships
+      buildspec: codebuild_specs/run_e2e_tests.yml
+      env:
+        compute-type: BUILD_GENERAL1_MEDIUM
+        variables:
+          TEST_SUITE: >-
+            src/__tests__/add-resources.test.ts|src/__tests__/base-cdk.test.ts|src/__tests__/custom-logic.test.ts|src/__tests__/relationships.test.ts
```

Comment:
This will probably look for these tests within the `amplify-e2e-tests` package. You need to add a different buildspec or update the `run_e2e_tests.yml` to accept tests directory.

Reply:
Yeah, you're right, I'll just create a different build target.